bootstrap/bootstrap.go

```diff
@@ -59,12 +58,43 @@ const (
 	apiTypeSignin  = "signin"
 )
 
-// bootstrap is a data structure to hold configuration required to start
+// Defaults.
+const (
+	DefaultIdentifierClientPath = "./identifier-webapp"
+	DefaultSigningKeyID         = "default"
+	DefaultSigningKeyBits       = 2048
+)
+
+//Stringly typed application config, represents the user accessible config params
```

Comment:
I suppose comments like these do not pass the linting. I know that `make lint` currently does produce quite some errors, but make sure at the very least that new code is passing, especially comments on exported stuff.